Frontend/src/common/api/endpoints.ts

```diff
@@ -53,6 +58,18 @@ export const rentBike = (bikeId: string) =>
     data: { id: bikeId },
   });
 
+export const reserveBike = (bikeId: string) =>
+  apiWithAuthConnection<Bike>(`${API}/bikes/reserved`, {
```

Comment:
Rowery zwracane przez `POST /bikes/reserved` oraz `GET /bikes/reserved` są inne niż ten model - zawierają informacje o rezerwacji (typ ReservedBike w RAMLu)

Reply:
Podmieniłem, ale nie wiem czy takie coś zadziała :D

Teraz coś jest nie tak, na backendzie mogę w nieskończoność wypożyczać dany rower i ciągle zwraca success, ale status roweru nigdy nie zmienia się na 'Reserved'. Dodałem więc coś takiego 
![image](https://user-images.githubusercontent.com/79532091/118561047-be2d7a80-b76a-11eb-930b-51427543da0f.png)

Jest teraz jeszcze taki problem że pobieranie rowerów na stacji nie zwraca rowerów które są wypożyczone przez obecnego użytkownika, a chyba powinno.

Reply:
Masz na myśli zarezerwowane? Nie wiem czy powinno szczerze mówiąc

Reply:
A mamy jakiś inny sposób na pobieranie wypożyczonych rowerów na danej stacji? Po prostu pobranie listy wszystkich wypożyczonych i sprawdzenie które z nich są na tej stacji na frontendzie?

Reply:
Jeśli są wypożyczone to nie są na żadnej stacji. Jeśli masz na myśli zarezerwowane, to swoje zarezerwowane widzisz w oddzielnej zakładce i tam masz przycisk do wypożyczenia, więc imo nie potrzebujemy ich na liście rowerów na stacji

Reply:
Poza tym wydaje mi się, że ustalone zostało że GET /stations/{id}/bikes dla usera zwraca listę dostępnych rowerów na danej stacji - zarezerwowany rower nie jest dostępny

Reply:
![image](https://user-images.githubusercontent.com/24509819/118563332-a8ba4f80-b76e-11eb-8efb-6be19fd995ff.png)

Reply:
Tak, chodziło mi o zarezerwowane ;)

Okej, masz rację, myślałem że będziemy chcieli mieć coś takiego
![image](https://user-images.githubusercontent.com/79532091/118563416-c982a500-b76e-11eb-815a-64d331fac2a5.png)

Ale zakładka Reserved w pełni wystarczy